levels/house_ceremony/maps/mainhall.json

```diff
@@ -1821,14 +1897,10 @@
         }, 
         {
          "firstgid":16317,
-         "source":"..\/..\/..\/..\/.bundled\/tilesets\/TwilioQuestObjects.json"
-        }, 
-        {
-         "firstgid":17333,
          "source":"..\/..\/..\/tilesets\/ApiAcademyObjects.json"
```

Comment:
It doesn't look like this is referencing a .bundled tileset, which **could** lead to issues down the line when we publish the extension. Let's keep an eye on this in case we run into any trouble!